docs/file-service/FileUpdateTransaction.md

```diff
@@ -0,0 +1,171 @@
+|---------|-----------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Updates only expiration time with proper payer signature                                     | Only expirationTime set with payer signature                                                                                        | The file update succeeds (no file key signatures required).                                                                      | N                 |
+| 2       | Updates keys without all required file key signatures                                        | keys=[<VALID_KEY>] without all file key signatures                                                                                  | The file update fails with `INVALID_SIGNATURE`.                                                                                  | N                 |
+| 3       | Updates contents without all required file key signatures                                    | contents="new content" without all file key signatures                                                                              | The file update fails with `INVALID_SIGNATURE`.                                                                                  | N                 |
+| 4       | Updates memo without all required file key signatures                                        | memo="new memo" without all file key signatures                                                                                     | The file update fails with `INVALID_SIGNATURE`.                                                                                  | N                 |
+| 5       | Updates multiple fields with all required file key signatures                                | keys, contents, memo all set with all file key signatures                                                                           | The file update succeeds.                                                                                                         | N                 |
+| 6       | Updates immutable file (empty key list) with non-expiration fields                           | keys=[], contents="new content"                                                                                                     | The file update fails with appropriate error.                                                                                    | N                 |
```

Comment:
Input column should only contain the input for each variable. I think if we need to add extra explanation to test, we should add an "Additional Notes" column or something to the table